### DISCUSSION TITLE
Enforce to comment why eslint-ignore was used

### DISCUSSION THREAD

**Discussion by kilian-goetz** [eslint/eslint#16695] [Ideas] [answered]
Hi there ✋,

I have done some searches on, and I don't find other discussion about. I think it could be great to implement a rule inside ESLint to obligate people to write a comment after an `// eslint-disable-next-line`. This will compel developer to justify the ignore line to avoid comments in pull request and next maintainer to return to GitHub closed pull requests 😃. 

Kind regards,
Kilian GOËTZ.

**Comment by mdjermanovic** [comment, marked as answer]
For this purpose, you can use the `eslint-comments/require-description` rule from [eslint-plugin-eslint-comments](https://www.npmjs.com/package/eslint-plugin-eslint-comments).

https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/require-description.html

**Reply by kilian-goetz** [reply]
Thanks, I don't know that's already exist 😄.
